catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/operation/PostgreSqlTableOperations.java

```diff
@@ -58,6 +60,62 @@ public void initialize(
         "The `jdbc-database` configuration item is mandatory in PostgreSQL.");
   }
 
+  @Override
+  public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
```

Comment:
Why haven't we implemented this logic in the superclass

Reply:
Because PG and MySQL are different here, only PG has the problem mentioned.